assets/scss/components/_references.scss

```diff
@@ -0,0 +1,58 @@
+.references {
+    &__item {
```

Comment:
Style-wise, I think these might look nice if they were inline-blocks of 50% width (side by side), will need to give it a try later